mergify_engine/web/simulator.py

```diff
@@ -141,6 +141,17 @@ async def simulator(request: requests.Request) -> responses.JSONResponse:
     else:
         title, summary = ("The configuration is valid", None)
 
+    pull_request_rules_conditions = []
```

Comment:
Any reason not to write that as a list comprehension?

Reply:
you are too fast for reviewing ;)